WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/SupportForm/SupportFormMetadataProvider.swift

```diff
@@ -204,6 +204,7 @@ private extension SupportFormMetadataProvider {
             }
         }()
 
+        // TODO: evaluate to not send anymore this information since this is deprecated and will not be replaced in the future by Apple.
```

Comment:
Just chiming in, I believe these can be removed as well. I've checked with mobile support folks and there's a common agreement that these are _never_ used in order to help with ticket resolution: p1709606144879569-slack-C03URUK5C

You can do it in this PR, or separately, whatever works best for you.